.lighthouse/profiles/desktop.js

```diff
@@ -33,13 +33,13 @@ module.exports = {
               },
               {
                 metric: 'speed-index',
-                budget: 300 // 0.3s
+                budget: 500 // 0.5s
```

Comment:
These are more realistic budgets.

Reply:
Discussed this with @mshick on slack, adding context here for posterity:

These budgets are being exceeded on the current main (b574b83217735720e1b773061584758f75f4ad19). As you can see from this screenshot taken from [a recent lighthouse report](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1710860981493-53474.report.html):
 
<img width="814" alt="Screen Shot 2024-03-21 at 3 21 55 PM" src="https://github.com/takeshape/penny/assets/428120/b1962e19-4e34-48cf-b9f4-a5ee76f94a57">

My concern was that these budgets were being upped to accommodate for a size increase / performance decrease in the v14 upgrade which is not the case we.